### PR TITLE
Add `fit_line_to_points` function

### PR DESCRIPTION
**Pull request by user27182**
### Overview

Similar to `fit_plane_to_points` but for a line.

### CODE REVIEW

**Review by tkoyama010 — APPROVED** [review]
LGTM. Awesome!

File: pyvista/core/utilities/points.py [hunk 1]
```diff
@@ -346,6 +355,136 @@ def fit_plane_to_points(points, return_meta=False, resolution=10):
...
+        Fit a plane using the first two principal axes of the points.
+
+    principal_axes
+        Compute axes vectors which best fit a set of points.
+
+    Returns
+    -------
+    pyvista.PolyData
+        Line mesh.
+
```
Comment (user27182) [inline pyvista/core/utilities/points.py:400]:
```suggestion

    numpy.ndarray
        Line length if ``return_meta=True``.

    numpy.ndarray
        Line direction (unit vector) if ``return_meta=True``.

```

**Review by tkoyama010 — APPROVED** [review]

File: pyvista/core/utilities/points.py [hunk 2]
```diff
@@ -398,6 +398,12 @@ def fit_line_to_points(
     pyvista.PolyData
         Line mesh.
 
+    numpy.ndarray
```
Comment (user27182) [inline pyvista/core/utilities/points.py:401]: Whoops, this should be a float, sorry about that!

**Review by tkoyama010 — DISMISSED** [review]
I will enable auto merge after 24 hours if there is no other comment :)

**Review by tkoyama010 — APPROVED** [review]

**Pull request merged**